neofetch

```diff
@@ -119,27 +119,44 @@ getdistro() {
     case "$os" in
         "Linux" )
             if type -p lsb_release >/dev/null 2>&1; then
+                distro="$(lsb_release -sd 2>/dev/null)"
```

Comment:
Change the case to this, otherwise `lsb_release` is run twice when `on` and `tiny` are used.

```sh
case "$distro_shorthand" in
    "on") distro="$(lsb_release -sir 2>/dev/null)" ;;
    "tiny") distro="$(lsb_release -si 2>/dev/null)" ;;
    *) distro="$(lsb_release -sd 2>/dev/null)" ;;
esac
```